### PR TITLE
SAMPLE : part of a church stained glass' visit

### PR DESCRIPTION
**Pull request by vallhallalm**
We, [Sébastien Bacoup, Amal Chikhaou, Matéo Condat, Léo Kemplaire, Louis Magnier, Le Vu Long Tran], hereby grant to Hyperglosae maintainers the right to publish our contribution under the terms of any licenses the Free Software Foundation classifies as Free Software Licenses.

### CODE REVIEW

File: samples/eglise-st-jean.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "_id":"ejrbzoefkznfkpz",
+    "dc_title":"Eglise St-Jean-au-Marché, Troyes",
+    "dc_creator": "Louis Magnie",
+    "dc_issued": "2016-07-02",
+    "text": "![](https://steatite.utt.fr/optimized/7b0006e98ec564021778cf585cae3adec07f4389)",
+    "dc_license": "https://creativecommons.org/licenses/by-sa/4.0/"
```
Comment (benel) [inline samples/eglise-st-jean.json:7]: The license is unknown. You should let it undefined (which means "All rights reserved").

File: samples/vitrail-1-1.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "_id": "ejrbzoefkznfkpz1-1",
+  "dc_title": "Vitrail 1, fragment 1",
+  "dc_isPartOf": "Eglise St-Jean-au-Marché, Troyes",
+  "dc_creator": "Louis Magnier",
```
Comment (benel) [inline samples/vitrail-1-1.json:5]: There is an ambiguity in this document about who is the author of what...

File: samples/vitrail-2.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "_id": "ejrbzoefkznfkpz2",
+  "dc_title": "Vitrail 2",
+  "dc_isPartOf": "Eglise St-Jean-au-Marché, Troyes",
+  "dc_creator": "Louis Magnier",
```
Comment (benel) [inline samples/vitrail-2.json:5]: Idem.

File: samples/vitrail1.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "_id": "ejrbzoefkznfkpz1",
+"dc_title": "Vitrail 1",
+"dc_isPartOf": "Eglise St-Jean-au-Marché, Troyes",
+"dc_creator": "Louis Magnier",
```
Comment (benel) [inline samples/vitrail1.json:5]: Idem.

**Review by benel — CHANGES_REQUESTED** [review]
Thank you very much for this contribution.

The use case is really interesting.

However by referring directly the picture in your own document, there is an ambiguity in who is  the author of what (i.e. you are not the author of the picture), which means that: 
- you don't use the main feature of the software,
- there is no more links between different visits that would include the same picture.

If your team need a work session with me to think about the possible network, I can help you.

**Comment by vallhallalm** [comment]
Normally I managed to do the changes requested @benel

**Comment by benel** [comment]
Thank you for your new commits @Hypertopic/histoire-de-l-art. 
However there is some confusion in this pull request (probably because it points to the `main` branch). 
I suggest you to close this pull request and create a new pull request on a new branch with the new samples you worked on last week.

**Pull request closed without merging**